ios/fastlane/verify/validate_dimensions.rb

```
@@ -0,0 +1,109 @@
+#!/usr/bin/env ruby
```

RuboCop (1.85.1)

[warning] Script file validate_dimensions.rb doesn't have execute permission.

(Lint/ScriptPermission)